lib/client-api/src/story_store.ts

```diff
@@ -152,10 +152,25 @@ export default class StoryStore {
     this._configuring = true;
   }
 
+  failConfiguring(error: Error) {
```

Comment:
We can just do this in `finishConfiguring` if `this._error` is set.

Reply:
ok, great!